### ISSUE TITLE
SQL comments with javadoc terminators break generated code

### ISSUE THREAD

**Issue by dsharinov** [jOOQ/jOOQ#4575]
In our Oracle database we have comments on columns like that:
```sql
comment on column VCS_CONSTANTS.c_airclosdflt
  is '/*"display=Air CLOS Dflt",filtered=N, "formLabelHint=Default for Air Request Class of Servis", "service=swdomain AIR_CLOS"*/';
```
We use JOOQ version 3.6.2 to generate code and according to Issue #3450 it should have been fixed. But what we get in Java code is this:
```java
	/**
	 * The column <code>VCS_CONSTANTS.C_AIRCLOSDFLT</code>. /*"display=Air CLOS Dflt",filtered=N, "formLabelHint=Default for Air Request Class of Servis", "service=swdomain AIR_CLOS"*/
	 */
	public final TableField<VcsConstantsRecord, String> C_AIRCLOSDFLT = createField("C_AIRCLOSDFLT", org.jooq.impl.SQLDataType.CHAR.length(1), this, "/*\"display=Air CLOS Dflt\",filtered=N, \"formLabelHint=Default for Air Request Class of Servis\", \"service=swdomain AIR_CLOS\"*/");
```
which does not compile.

**Comment by lukaseder** [comment]
Thank you very much for reporting. It looks as though the fix from #3450 is applied only to table comments, not to column comments. We'll look into this immediately.

**Comment by lukaseder** [comment]
This fix will be merged to 3.5.5 (#4585) and 3.6.4 (#4584). We're planning on releasing 3.6.4 hopefully next week, as we have another high priority fix to publish.
